server/src/test/java/org/elasticsearch/action/OriginalIndicesTests.java

```diff
@@ -43,15 +42,7 @@ public void testOriginalIndicesSerialization() throws IOException {
             OriginalIndices originalIndices2 = OriginalIndices.readOriginalIndices(in);
 
             assertThat(originalIndices2.indices(), equalTo(originalIndices.indices()));
-            // indices options are not equivalent when sent to an older version and re-read due
-            // to the addition of hidden indices as expand to hidden indices is always true when
-            // read from a prior version
-            if (out.getTransportVersion().onOrAfter(TransportVersions.V_7_7_0)
-                || originalIndices.indicesOptions().expandWildcardsHidden()) {
-                assertThat(originalIndices2.indicesOptions(), equalTo(originalIndices.indicesOptions()));
-            } else if (originalIndices.indicesOptions().expandWildcardsHidden()) {
-                assertThat(originalIndices2.indicesOptions(), equalTo(originalIndices.indicesOptions()));
-            }
+            assertThat(originalIndices2.indicesOptions(), equalTo(originalIndices.indicesOptions()));
```

Comment:
Isn't this missing `if (originalIndices.indicesOptions().expandWildcardsHidden())`?

Reply:
The first `onOrAfter` check is always true, which means it never evaluates the second `if` clause

Reply:
Oh yeah, sure! Sorry about that